test/libsolidity/syntaxTests/array/contract_index_access.sol

```diff
@@ -0,0 +1,8 @@
+contract C {
+    function f() view public {
+        C[0];
```

Comment:
Wasn't this an error in another test?

Reply:
library vs contract

Reply:
yea, at first I read this as `index accessing C`, not as a declaration of an array